javascripts/discourse/initializers/customize_new_topic_text.js

```diff
@@ -0,0 +1,85 @@
+import { withPluginApi } from "discourse/lib/plugin-api";
+import I18n from "I18n";
```

Comment:
The import has changed:

```suggestion
import I18n from "discourse-i18n";
```